stdlib/sqlite3/dbapi2.pyi

```diff
@@ -227,7 +227,7 @@ else:
 
 if sys.version_info < (3, 8):
     class Cache:
-        def __init__(self, *args: Incomplete, **kwargs: object) -> None: ...
+        def __init__(self, *args: Incomplete, **kwargs: Unused) -> None: ...
```

Comment:
I'm not sure the existing annotation for this is correct, but also: it's 3.7-only, so I'm not sure I care too much 😛

Reply:
Nor can I find "Cache" in the docs 🤷 
https://docs.python.org/3.7/library/sqlite3.html